sagemaker-clarify/time_series_deepar/time_series_deepar.ipynb

```diff
@@ -0,0 +1,2293 @@
+      "                                          'direction': 'chronological',\n",
+      "                                          'granularity': 'fine_grained',\n",
+      "                                          'num_samples': 9},\n",
+      "             'report': {'name': 'report', 'title': 'Analysis Report'}},\n",
+      " 'predictor': {'accept_type': 'application/json',\n",
+      "               'content_template': '{\"instances\": $records}',\n",
+      "               'content_type': 'application/json',\n",
+      "               'endpoint_name': 'DeepArTest-2024-04-09-17-56-24-856',\n",
+      "               'record_template': '{\"start\": $start_time, \"target\": $target_time_series, \"dynamic_feat\": $related_time_series, \"cat\": $static_covariates}',\n",
+      "               'time_series_predictor_config': {'forecast': 'predictions[*].mean'}},\n",
+      " 'time_series_data_config': {'dataset_format': 'timestamp_records',\n",
+      "                             'item_id': '[].item_id',\n",
+      "                             'related_time_series': ['[].dynamic_feature_1', '[].dynamic_feature_2', '[].dynamic_feature_3'],\n",
+      "                             'static_covariates': ['[].static_feature_1', '[].static_feature_2'],\n",
+      "                             'target_time_series': '[].target_value',\n",
+      "                             'timestamp': '[].timestamp'}}\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(\"./analysis_config.json\", \"r\") as analyis_config_file:\n",
+    "    analysis_config = json.load(analyis_config_file)\n",
+    "    # mercury.JSON(analysis_config, level = 3)\n",
+    "    config_printer = pprint.PrettyPrinter(width=200, compact=False)\n",
+    "    config_printer.pprint(analysis_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8bba3978-17c9-4685-ae72-04b61c9510a7",
+   "metadata": {},
+   "source": [
+    "## Explainability Results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4bfc1be6-9de4-4c62-8cea-4f070b52754b",
+   "metadata": {},
+   "source": [
+    "### Retrieve Results From s3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "id": "b083a9df-17f6-4e0e-bccc-2f8585880744",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "full_result_path = f\"deepar/output/asymmetric_shapley_value/{granularity}_{direction}/out.jsonl\"\n",
+    "\n",
+    "s3.download_file(bucket_name, full_result_path, \"results.jsonl\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee194253-17db-45c2-b6c9-721d1ca1336a",
+   "metadata": {},
+   "source": [
+    "### Display Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "ad0142ad-f709-4cb6-9b79-4ffa482341ca",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
```

Comment:
As far as I can tell it doesn't look like this output using `mercury` is being displayed by GitHub. Perhaps it'd be best to swap to pretty printer?